TShida951/kpi_management_front
# [KM-2][Users] [FE] Implement Component Ticket Label

Create Component Ticket Label

- Todo, Backlog: background grey
- Done, Complete, Release: background green
- Others: background blue

Step to test:

1. Access userlists page.
2. Click on an username to access user-detail page.
3. Check if styling, label for ticket's status and ticket's type is display or not
